style.css

```diff
@@ -4188,22 +4211,14 @@ ul {
   margin: 0 5px;
 }
 
-/* Non-latin search results highlight */
-/* Add a yellow background for Chinese */
-html[lang|="zh"] .search-result-description em {
+/* By default use bold instead of italic to highlight */
+.search-results-description em {
   font-style: normal;
-  background: yellow;
+  font-weight: bold;
 }
 
-/* Use bold to highlight for the rest of supported non-latin languages */
-html[lang|="ar"] .search-result-description em,
-html[lang|="bg"] .search-result-description em,
-html[lang|="el"] .search-result-description em,
-html[lang|="he"] .search-result-description em,
-html[lang|="hi"] .search-result-description em,
-html[lang|="ko"] .search-result-description em,
-html[lang|="ja"] .search-result-description em,
-html[lang|="ru"] .search-result-description em,
-html[lang|="th"] .search-result-description em {
-  font-style: bold;
```

Comment:
Are these no longer needed?

Reply:
we now set `font-weight` to `bold` for all locales (except Chinese). so we don't need these any more